frontend/components/CourseEvaluation/SectionTabs.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { useSWRAuth } from '@/components/hooks/useSWRAuth';
+import {
+  API_ENDPOINT,
+  CourseEvaluationListEntry,
+  DEFAULT_COURSE_EVALUATION_LIST_ENTRY,
+} from 'utils/api';
+import CustomTheme from '../utils/CustomTheme';
+
+import Overview from './Overview';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+type Props = {
+  id: string;
```

Comment:
be specific... what type of id ... i would expect this to be called the `courseEvaluationId`...

I understand that the folder name is `CourseEvaluation` however, later on we can have `reviewId` as part of the components.

The only time it is acceptable to just say ID is if it prepended by something... eg. `course_evaluation.id`  (this is in the backend which is why it is snake case and that it is an object that shows the context at which the `id` field refers to